pkg/machines/components/networks/createNetworkDialog.jsx

```diff
@@ -43,17 +43,25 @@ const ConnectionRow = ({ connectionName }) => {
     );
 };
 
-function validateParams(dialogValues) {
+function validateParams(dialogValues, networks) {
     const validationFailed = {};
 
     if (isEmpty(dialogValues.name.trim()))
         validationFailed.name = _("Name should not be empty");
 
     if (dialogValues.ip === "IPv4 only" || dialogValues.ip === "IPv4 and IPv6") {
-        if (isEmpty(dialogValues.ipv4.trim()))
+        if (isEmpty(dialogValues.ipv4.trim())) {
             validationFailed.ipv4 = _("IPv4 network should not be empty");
-        else if (!utils.validateIpv4(dialogValues.ipv4))
+        } else if (!utils.validateIpv4(dialogValues.ipv4)) {
             validationFailed.ipv4 = _("Invalid IPv4 address");
+        } else {
+            const net = networks.some(net => net.ip[0] &&
+                                      net.ip[0].family === "ipv4" &&
+                                      net.ip[0].netmask === dialogValues.netmask &&
+                                      utils.isIpv4InNetwork(dialogValues.ipv4, dialogValues.netmask, net.ip[0].netmasknetwork.ipv4.address));
+            if (net)
+                validationFailed.ipv4 = cockpit.format(_("IPv4 address is already used by network '$0'"), net.name);
```

Comment:
@KKoukiou Yes, the if the other network is not running, they should be allowed to create a network with same address. So I will add this a condition that only if network is active the error message gets rendered.